Corona-Warn-App/schemas/de.rki.coronawarnapp.presencetracing.risk.storage.PresenceTracingRiskDatabase/1.json

```diff
@@ -69,6 +69,12 @@
             "columnName": "riskStateCode",
             "affinity": "INTEGER",
             "notNull": true
+          },
+          {
+            "fieldPath": "calculatedFromMillis",
```

Comment:
Do we need the changes here from `1.json`? We are migrating to `2.json` anyway?

Reply:
No this file was generated after I added the field and before I changed the version. I will remove it.